jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/HttpConnectionFactory.java

```diff
@@ -27,10 +28,9 @@
  * {@link HttpConnection}s are configured by a {@link HttpConfiguration} instance that is either created by
  * default or passed in to the constructor.
  */
-public class HttpConnectionFactory extends AbstractConnectionFactory implements HttpConfiguration.ConnectionFactory
+public class HttpConnectionFactory extends AbstractConnectionFactory implements HttpConfiguration.ConnectionFactory, ConnectionFactory.Configuring
```

Comment:
With move to HttpConfiguration for ComplianceViolation.Listener config, this class can be reverted to base.